app/assets/javascripts/3dbio_viewer/src/webapp/components/RootViewerContents.tsx

```diff
@@ -47,9 +48,21 @@ export const RootViewerContents: React.FC<RootViewerContentsProps> = React.memo(
     const toggleToolbarExpanded = React.useCallback(
         (_e: React.SyntheticEvent, data: ResizeCallbackData) => {
             setToolbarExpanded(data.size.width >= 520);
-            setViewerSelectorExpanded(window.innerWidth - data.size.width >= 725);
+            setViewerSelectorExpanded(innerWidth - data.size.width >= 725);
         },
-        []
+        [setToolbarExpanded, setViewerSelectorExpanded, innerWidth]
+    );
```

Comment:
Sorry, I hadn't seen your question regarding the viewport changes. Indeed, you cannot simply add a window prop here, that's not something React can react to, you have to listen to the resize event. Use a hook like this: https://stackoverflow.com/a/19014495/188031

If you see that is not as simple as using this hook, let me know.